yarn.lock

```diff
@@ -0,0 +1,7243 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do we really need locked versions? Couldn't we trust SemVer?

Reply:
It's risky for npm dependencies. Not all frontend libs follow SemVer

Reply:
Yes I would prefer sticking to the `yarn.lock` file too 👍

Reply:
I agree

Reply:
Well, let's keep them.